pytorch/inference/docker/1.11/py3/Dockerfile.cpu

```diff
@@ -2,6 +2,10 @@ ARG PYTHON=python3
 ARG PYTHON_VERSION=3.8.13
```

Comment:
Can we revert the changes for PT 1.11 and 1.12? These images are past due as per our support policy https://aws.amazon.com/releasenotes/dlc-support-policy/

Reply:
Done